### PR TITLE
Let wallet importmulti RPC accept labels for standard scriptPubKeys

### PR DESCRIPTION
**Pull request by ryanofsky**
Allow importmulti RPC to apply address labels when importing standard scriptPubKeys. This makes the importmulti RPC less finnicky about import formats and also simpler internally.

### CODE REVIEW

**Comment by kameroni** [comment]
Bonjour

**Comment by ryanofsky** [comment]
Rebased ef25b9a203002799fe2f58fb566bc58ff8c89648 -> 4f1abbd7390d60eb96c473031018ca9ab402e10c ([script-label.3](https://github.com/ryanofsky/bitcoin/commits/pr/script-label.3) -> [script-label.4](https://github.com/ryanofsky/bitcoin/commits/pr/script-label.4)) because of conflicts with recent changes to import-rescan.py and importmulti.py.

**Comment by ryanofsky** [comment]
Rebased 4f1abbd7390d60eb96c473031018ca9ab402e10c -> 21c179afa682721c6484e4ec468146ce513cdec4 ([pr/script-label.4](https://github.com/ryanofsky/bitcoin/commits/pr/script-label.4) -> [pr/script-label.5](https://github.com/ryanofsky/bitcoin/commits/pr/script-label.5)) because of conflicts with pwalletMain renames in #9775.

File: test/functional/wallet_import_rescan.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def run_test(self):
 
         # Create new transactions sending to each address.
         for i, variant in enumerate(IMPORT_VARIANTS):
-            variant.sent_amount = 10 - (2 * i + 1) / 8.0
+            variant.sent_amount = 1 - (2 * i + 1) / 128
```
Comment (jamesob) [inline test/functional/wallet_import_rescan.py]: Curious what's prompting these value changes.

**Review by jamesob — COMMENTED** [review]
utACK https://github.com/bitcoin/bitcoin/pull/9332/commits/98ea64cf232c34d4b1aebe738b3956191667cd76

I don't see anything in the tests exercising the application of a label to a scriptPubKey import; if we're indeed lacking that I'm happy to either add a commit here or file a follow-up PR.

File: test/functional/wallet_import_rescan.py (hunk 1 above)
Comment (ryanofsky) [reply test/functional/wallet_import_rescan.py]: > Curious what's prompting these value changes.

Change above is This is just reducing

**Review by ryanofsky — COMMENTED** [review]
> I don't see anything in the tests exercising the application of a label to a scriptPubKey import; if we're indeed lacking that I'm happy to either add a commit here or file a follow-up PR.

This is tested in `wallet_import_rescan.py` on line 61 when the call type is `Call.multiscript`. (If you revert the code changes and run the test you should see this case fail.)

**Comment by sipa** [comment]
utACK 98ea64cf232c34d4b1aebe738b3956191667cd76. Looks like a nice simplification too.

**Comment by laanwj** [comment]
utACK 98ea64cf232c34d4b1aebe738b3956191667cd76

**Pull request merged**